build/chunk-module-manifest-plugin.js

```diff
@@ -22,8 +22,18 @@ class ChunkModuleManifestPlugin {
           const chunkIdsByFile = new Map();
           chunks.forEach(c => {
             const chunkId = c.id;
-            const files = Array.from(c.modulesIterable, m => m.resource);
-            files.forEach(path => {
+
+            let additionalFileDeps = [];
+            const files = Array.from(c.modulesIterable, m => {
+              if (m.rootModule && m.rootModule.resource) {
+                additionalFileDeps = additionalFileDeps.concat(
```

Comment:
I'm not sure this is the correct fix. It might be, but I was able to fix this issue with a different set of changes. Let me put up my PR and we can compare.

Reply:
I think both will probably work, but yours is cleaner, thanks!